lib/graphql-ast.explorer.ts

```diff
@@ -184,7 +184,21 @@ export class GraphQLAstExplorer {
       returnType: `${type} | Promise<${type}>`,
       parameters: this.getFunctionParameters(
         (item as FieldDefinitionNode).arguments,
-      ),
+      ).sort((a, b) => {
```

Comment:
perhaps we should extract this comparator to a separate function + simplify it a little bit, for example this
```typescript
else if (a.hasQuestionToken) {
    return 1;
} else {
    return -1;
}
```
to:
```typescript
return a.hasQuestionToken ? 1 : -1;
```

Reply:
Done. I don't have a development environment around now, so I did the changes in github...